bin/main.ml

```diff
@@ -869,6 +870,32 @@ let subst =
          )
   , Term.info "subst" ~doc ~man)
 
+let utop =
+  let doc = "Load library in utop" in
+  let man = [ (* TODO *) ] in
+  let go common dir args =
+    let utop_target = dir |> Path.of_string |> Utop.target |> Path.to_string in
+    set_common common ~targets:[utop_target];
+    (* We must wait for other exit hooks to finish before forking. This is
+       necessary to make sure the trace file is dumped before we fork. *)
+    let utop_path = ref None in
+    at_exit (fun () -> Option.iter !utop_path ~f:(fun p ->
+      let p = Path.to_string p in
+      Unix.execv p (Array.of_list (p :: args))
+    ));
+    let log = Log.create () in
+    Future.Scheduler.go ~log
+      (Main.setup ~log common >>= fun setup ->
+       let targets = resolve_targets ~log common setup [utop_target] in
```

Comment:
If we have multiple build contexts defined, we are going to build several utop executables and run only one. This doesn't seem right to me. When `targets` has more than 1 element, we should fail and ask the user to pass a `--context <name>`  argument to select a particular build context